src/strings/en-us.json

```diff
@@ -1728,5 +1728,20 @@
     "MachineTranslated": "Machine Translated",
     "ForeignPartsOnly": "Forced/Foreign parts only",
     "HearingImpairedShort": "HI/SDH",
-    "LabelIsHearingImpaired": "For hearing impaired (SDH)"
+    "LabelIsHearingImpaired": "For hearing impaired (SDH)",
+    "Author": "Author",
+    "Authors": "Authors",
+    "Colorist": "Colorist",
+    "CoverArtist": "Cover artist",
+    "Creator": "Creator",
+    "Creators": "Creators",
+    "Editor": "Editor",
+    "Editors": "Editors",
+    "HeaderCreators": "Creators",
+    "Illustrator": "Illustrator",
+    "Inker": "Inker",
+    "Letterer": "Letterer",
+    "Penciller": "Penciller",
+    "Translator": "Translator",
+    "Translators": "Translators"
```

Comment:
I meant, put the new values to where they should be in alphabetical order in the whole file.
I believe the first part of the file is ordered correctly.